src/main/scala/beam/sim/config/MatSimBeamConfigBuilder.scala

```diff
@@ -34,10 +34,24 @@ class MatSimBeamConfigBuilder(beamConf: Config) {
 
   def buildMatSamConf() = {
     val matSimConfig = ConfigUtils.createConfig(Paths.get(beamConf.origin().url().toURI).getParent.toUri.toURL)
-    val paramSetClassCache = RefectionUtils.concreteClassesOfType[MatsimParameters].
-      collect { case clazz if RefectionUtils.isExtends(clazz, classOf[ConfigGroup]) =>
-        Try(clazz.newInstance()).toOption
-      }.flatten.map(paramSet => {
+    val maybeParameterSets = MatSimBeamConfigBuilder.concreteClassesOfType[MatsimParameters].
+      collect { case clazz if MatSimBeamConfigBuilder.isExtends(clazz, classOf[ConfigGroup]) =>
+        try {
+          Option(clazz.newInstance())
+        } catch {
+          case e: IllegalAccessException =>
+            logger.warn(s"Couldn't instantiate MatsimParameters  '${clazz.getCanonicalName}'. It doesn't have default public constructor.Falling back to setAccessible(). Cause : " + e.getMessage)
+            Try {
+              val c = clazz.getDeclaredConstructor()
+              c.setAccessible(true)
```

Comment:
Some of MatSim classes have protected constructor, thus we have to apply this workaround